cfme/tests/services/test_provision_stack.py

```diff
@@ -174,33 +182,35 @@ def _cleanup_vms():
     assert row.last_message.text == 'Service Provisioned Successfully'
 
 
-@pytest.mark.meta(blockers=[1221333])
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
 def test_reconfigure_service(setup_provider, provider, provisioning, dialog, catalog, request):
     """Tests stack provisioning
 
     Metadata:
         test_flag: provision
     """
-    dialog_name, template_name = dialog
+    dialog_name_new = "dialog_" + fauxfactory.gen_alphanumeric()
+    template_type = provisioning['stack_provisioning']['template_type']
+    template = OrchestrationTemplate(template_type=template_type,
+                                     template_name=fauxfactory.gen_alphanumeric())
+    template.delete_all_templates()
+    template.create(METHOD_TORSO + "           ")
```

Comment:
Template has a format , no comments can be added . 
I can try adding different templates but that will make the code very lengthy as each template is very long and the one i added was the shortest i could find. 
I could think of only this way to use same template for every test.